### DISCUSSION TITLE
Reverse (invert, flip) tab order: more recent on top and downwards

### DISCUSSION THREAD

**Discussion by MBS17** [mbnuqw/sidebery#1204] [Q&A]
Is there a way to reverse the order tabs appear on Tree? Having more recent tabs on top and going down from more recent tabs to older ones getting at the bottom? It seems to me a more natural navigational position for some people. Thanks.

**Comment by mbnuqw** [comment]
You can only change the position of the new tab to `panel start`/`first child` (in sidebery settings), but this won't change the direction of the tree branches (the child tab will always come after the parent tab).

**Reply by MBS17** [reply]
Thanks! I was actually trying a similar solution (or the same, IDK). I sorted my tabs reverse with Foxytabs, then went to Sidebery "Position of new tab" and set everything "Panel Start". Also trying to set everywhere something like when closing tab goes to previous tab. Kind of working like what I wanted. Mostly have to change how i perceive FF own tabs row in reverse, going from left to right.

PS: This seems to make some navigation easier, as you can go to last tab using keyboard shortcuts like ctrl+Number.
